test/unit/test-url-replacements.js

```diff
@@ -961,6 +961,14 @@ describes.sandboxed('UrlReplacements', {}, () => {
           delete eventListeners[eventType];
         }
       };
+      const timerMock = {
```

Comment:
While technically this is not necessary because we don't have a unit test for loadEventEnd. But I figure i'll leave it there : ) Let me know if you don't like it. Thanks

Reply:
test will break without such a mock right?

Reply:
It shouldn't because there's no test for loadEventEnd. I could create one if you find it necessary.

Reply:
I agree that I don't see it quite necessary for such a test , unless you can reproduce the issue in a unit test failure. (Doable, but requires a lot more mocking of the performanceTimingAPI).

And this should be easier to test with an integration test, as it is depending on browser API behavior.

I think we can remove this timerMock for now, as it only brings confusion.

Reply:
done